sympy/stats/frv_types.py

```diff
@@ -91,6 +85,12 @@ def FiniteRV(name, density):
     >>> P(X >= 2)
     0.700000000000000
 
+    Parameters
+    ==========
+
+    density : A dict
+        Dictionary conatining the pdf of finite distribution
```

Comment:
Possibly out of scope here but a dict is fundamentally discrete to cannot be a "pdf". It could be a "PMF".